### ISSUE TITLE
Portable Intellij IDEA Ultimate/Android Studio installations not being recognised by "flutter doctor"

### ISSUE THREAD

**Issue by diego-ar** [flutter/flutter#61304]
I used to have a portable development environment, having a IntelliJ IDEA Ultimate installed in a external SSD drive (https://portapps.io/app/intellij-idea-ultimate-portable/).

The problem is when I run "flutter doctor -v", the flutter doesn't recognise the portable version (and I initially understand, because is not in the default directory). But the problem is that I haven't a option to do manually the search. The command "flutter config --android-studio-dir" doesn't work even if I use a Android Studio portable.

I have everything configured correctly in my external drive, here is my current "flutter doctor -v" output:

`[√] Flutter (Channel stable, v1.17.5, on Microsoft Windows [Version 10.0.18363.900], locale en-GB)
    • Flutter version 1.17.5 at Z:\SDK\Flutter
    • Framework revision 8af6b2f038 (10 days ago), 2020-06-30 12:53:55 -0700
    • Engine revision ee76268252
    • Dart version 2.8.4

[√] Android toolchain - develop for Android devices (Android SDK version 30.0.1)
    • Android SDK at Z:\SDK\Android
    • Platform android-30, build-tools 30.0.1
    • ANDROID_HOME = Z:\SDK\Android
    • Java binary at: Z:\SDK\OpenJDK\bin\java
    • Java version OpenJDK Runtime Environment (AdoptOpenJDK)(build 1.8.0_252-b09)
    • All Android licenses accepted.

[!] Android Studio (not installed)
    • Android Studio not found; download from https://developer.android.com/studio/index.html
      (or visit https://flutter.dev/docs/get-started/install/windows#android-setup for detailed instructions).

[!] Connected device
    ! No devices available

! Doctor found issues in 2 categories.`

**Comment by jmagman** [comment]
@christopherfujino I'll let you dupe this out 😄

**Comment by christopherfujino** [comment]
@diego-ar what happens when you try `flutter config --android-studio-dir <path to android studio>`?

**Comment by diego-ar** [comment]
> @diego-ar what happens when you try `flutter config --android-studio-dir <path to android studio>`?

Nothing, the flutter doctor doesn't recognise my Android Studio. I'm using the https://portapps.io Android Studio Portable App.

**Comment by christopherfujino** [comment]
Ahh, I see, so portapps is an application bundling technology?

**Comment by diego-ar** [comment]
> Ahh, I see, so portapps is an application bundling technology?

Yes, is like the PortableApps service. The Android Studio and the IntelliJ IDEA Ultimate works very well.
